src/main/java/com/artipie/api/ManageRepoSettings.java

```diff
@@ -50,18 +51,41 @@ public Collection<String> list(final String uname) {
     }
 
     @Override
-    public boolean exists(final String name) {
-        throw ManageRepoSettings.NOT_IMPLEMENTED;
+    public boolean exists(final String rname) {
+        return this.exists(rname, ext -> ext.key(rname));
     }
 
     @Override
-    public JsonStructure value(final String name) {
-        return Json.createObjectBuilder().add("type", "maven").add("storage", "def").build();
+    public boolean exists(final String uname, final String rname) {
+        return this.exists(rname, ext -> ext.key(String.format("%s/%s", uname, rname)));
     }
 
     @Override
-    public void save(final String name, final JsonStructure value) {
-        throw ManageRepoSettings.NOT_IMPLEMENTED;
+    public JsonObject value(final String rname) {
+        final byte[] value = this.asto.value(ManageRepoSettings.key(rname));
```

Comment:
variable can be inlined, redundant variables are not welcome in Artipie 
https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html